src/api/database/find.rst

```diff
@@ -55,8 +55,7 @@
         returns an opaque string under the ``bookmark`` key that can then be
         passed back in a query to get the next page of results. If any part of
         the query other than ``bookmark`` changes between requests, the results
-        are undefined. *Optional, default: null  Only for indexes of type
-        ``text``.*
+        are undefined. *Optional, default: null
```

Comment:
is it just the `selector` that should be consistent between calls to `_find`? If `limit` or `skip` are defined when passing a bookmark, won't that generate unexpected results?